conftest.py

```diff
@@ -42,7 +42,7 @@ def get_task_ids(user: str=None):
 
 def get_task_logs_for_id(task_id: str,  task_file: str='stdout', lines: int=1000000):
```

Comment:
"Naming things is hard"? I'd expect a method name "get_X" to return X or throw. The only reason I'm commenting on this is b/c removing `--completed` task ids above and task logs w/i this method are changes to the semantic meaning of the method, BUT given that the method is already:
1. not named very well according to its semantic
1. localized in effect b/c the "got" thing is actually emitted to stdout

The change in semantic to not include completed items is good.

Reply:
Totally agree. will consider that the next time something near this gets touched